lld/ELF/InputSection.h

```diff
@@ -48,7 +48,7 @@ template <class ELFT> struct RelsOrRelas {
 // sections.
 class SectionBase {
 public:
-  enum Kind { Regular, Synthetic, EHFrame, Merge, Output };
+  enum Kind { Regular, Synthetic, EHFrame, Merge, Output, Spill };
```

Comment:
`Regular, Synthetic, Spill, EHFrame, Merge, Output` may be more efficient.

Reply:
Done.